stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/StreamVideoBuilder.kt

```diff
@@ -102,6 +103,7 @@ public class StreamVideoBuilder @JvmOverloads constructor(
     private val crashOnMissingPermission: Boolean = false,
     private val permissionCheck: StreamPermissionCheck = DefaultStreamPermissionCheck(),
     private val audioUsage: Int = defaultAudioUsage,
+    private val appName: String? = null,
```

Comment:
`appName` is optional. Should we make it mandatory?

Reply:
Can the packagename be the default if not supplied?

Reply:
I added the packageName as a fallback. The header when `appName` is not provided looks like: `stream-video-android-1.0.13|os=Android 14|api_version=34|device_vendor=samsung|device_model=SM-A155F|app_name=io.getstream.video.android.debug`